### PR TITLE
feat: image and json bidirectional channels integrated

### PR DESCRIPTION
**Pull request by klepo**
- compatibility with era-5g-interface 0.7.0, era-5g-client 0.8.0, and era-5g-server 0.1.0
- use of mmcv>=2.1.0 and mmdet>=3.2.0
- new Dockerfiles for mmcv and standalone version
- use of CUDA images in GPU Dockerfiles due to Kubernetes

### CODE REVIEW

File: src/docker/ros2_object_detection/Dockerfile [hunk 1]
```diff
@@ -9,8 +9,6 @@ RUN apt-get update && apt-get install -y curl gnupg2 lsb-release software-proper
 RUN curl -sSL https://raw.githubusercontent.com/ros/rosdistro/master/ros.key -o /usr/share/keyrings/ros-archive-keyring.gpg
 RUN echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/ros-archive-keyring.gpg] http://packages.ros.org/ros2/ubuntu $(. /etc/os-release && echo $UBUNTU_CODENAME) main" | tee /etc/apt/sources.list.d/ros2.list > /dev/null
 
-RUN wget -c https://download.openmmlab.com/mmdetection/v2.0/yolo/yolov3_mobilenetv2_320_300e_coco/yolov3_mobilenetv2_320_300e_coco_20210719_215349-d18dff72.pth -O "/root/mmdetection/configs/yolo/yolov3_mobilenetv2_320_300e_coco_20210719_215349-d18dff72.pth"
```
Comment (Kapim) [inline src/docker/ros2_object_detection/Dockerfile:12]: How is the pth file downloaded?

File: src/python/README.md [hunk 2]
```diff
@@ -8,27 +8,20 @@ cd Reference-NetApp
 python3 -m venv env
 source env/bin/activate
 
-cd
-cp -r /srv/opencv ~
-pip3 install ~/opencv/*.whl
+# Install interface from era-5g-interface repository
+pip3 install era-5g-interface
```
Comment (Kapim) [inline src/python/README.md:12]: Is this necessary? It should be in the requirements.txt with a correct version

**Review by Kapim — APPROVED** [review]

File: src/python/README.md (hunk 2 above)
Comment (klepo) [reply src/python/README.md:12]: It is not necessary... you're right

File: src/docker/ros2_object_detection/Dockerfile (hunk 1 above)
Comment (klepo) [reply src/docker/ros2_object_detection/Dockerfile:12]: Through the mim as in the README. 
E.g.: 
mim download mmdet --config yolov3_mobilenetv2_8xb24-320-300e_coco --dest .
The mim tool will download both files, *.py and *.pth

Comment (Kapim) [reply src/docker/ros2_object_detection/Dockerfile:12]: Ok, but it is not in the Dockerfile...

Comment (klepo) [reply src/docker/ros2_object_detection/Dockerfile:12]: It is:
[src/docker/era_5g_netapp_base_mmcv/gpu.Dockerfile](https://github.com/5G-ERA/Reference-NetApp/pull/59/files#diff-9d32eda40b113e2aab6620a43c37678cd29949540d4ffe5a33a61573c1e1f0c8)
and:
https://github.com/5G-ERA/Reference-NetApp/pull/59/files#diff-70ec977244601c98db30beeac823949927f99cdcd40a297aa341c102443a4f1b

**Pull request merged**